src/commonMain/kotlin/character/Spec.kt

```diff
@@ -12,7 +12,7 @@ abstract class Spec {
             Triple("strength", Stats(strength = 50), 50.0),
             Triple("agility", Stats(agility = 50), 50.0),
             Triple("meleeCritRating", Stats(meleeCritRating = 5.0 * Rating.critPerPct), 5.0 * Rating.critPerPct),
-            Triple("physicalHitRating", Stats(physicalHitRating = 2.0 * Rating.physicalHitPerPct), 2.0 * Rating.physicalHitPerPct),
+            Triple("physicalHitRating", Stats(physicalHitRating = -5.0 * Rating.physicalHitPerPct), -5.0 * Rating.physicalHitPerPct),
```

Comment:
This is actually a lot cleaner!  However, we would need to update all entries for `physicalHitRating`, `expertiseRating`, and `spellHitRating` to use the negative modifiers.  Those were all the same idea.

Reply:
Done